### PR TITLE
freesolo the transmission suggest box

### LINKED ISSUE #328: When selecting 'other' no free text option offered
**Describe the bug**
When selecting 'other' in fields: Transmission and Place of transmission no free text option is offered.

**To Reproduce**
1. Go to 'Route of transmission'
2. Click on 'select other'

**Expected behavior**
Free text option offered.

**Screenshots**

**Environment (please complete the following information):**
 - OS
 - chrome
 -  83.0.4103.116

### PR DESCRIPTION
**Pull request by attwad**
Fixes #328

also fixes a flaky pagination test (my bad) that was running fine or not depending on the hour of the day /sigh.

### CODE REVIEW

**Comment by attwad** [comment]
[Finish new case manual entry page](https://trello.com/c/haOgIswr/246-finish-new-case-manual-entry-page)

File: suggest/route_of_transmission.txt [hunk 1]
```diff
@@ -6,6 +6,5 @@ Oral
 Direct contact
 Vertical (Parent to child)
 Iatrogenic
-Vector borne
-Other
+Vector bornOther
```
Comment (khmoran) [inline suggest/route_of_transmission.txt:9]: This diff looks strange - intentional?

Comment (attwad) [reply suggest/route_of_transmission.txt:9]: ah good catch, fixed

**Comment by attwad** [comment]
> Just an fyi I'm technically out today but it turns out GitHub mobile is pretty decent for reviewing prs

oh no sorry, I thought you were in today but not yesterday... my bad, thanks for reviewing anyway~ won't send anything else enjoy your day :)

**Pull request merged**